templates/taxi/manufacturer_list.html

```diff
@@ -23,11 +26,19 @@ <h1>Manufacturer List
               <td>
                   {{ manufacturer.country }}
               </td>
+              <td>
+                <a href="{% url "taxi:manufacturer-update" manufacturer.pk %}" class="btn btn-outline-primary change-link">Update</a>
```

Comment:
Consider using single quotes consistently in the `{% url %}` tag for the 'manufacturer-update' link to match the style used elsewhere in the template, such as in line 6.